### ISSUE TITLE
prohibit void attributes lower than other attributes

### ISSUE THREAD

**Issue by maxonfjvipon** [objectionary/lints#292]
Such XMIR is valid and correct
```xml
<o name="foo">
  <o base="∅" name="x"/>
  <o base="∅" name="y"/>
  <o base="Q.foo" name="z"/>
</o>
```

Such XMIR is not correct:
```xml
<o name="foo">
  <o base="∅" name="x"/>
  <o base="Q.foo" name="z"/>
  <o base="∅" name="y"/>
</o>
```

the same as:
```xml
<o name="foo">
  <o base="Q.foo" name="z"/>
  <o base="∅" name="x"/>
  <o base="∅" name="y"/>
</o>
```

In general - all void attributes must be higher than any other

**Comment by h1alexbel** [comment]
@maxonfjvipon assign me, please
